### PR TITLE
Emails can now be re-vaulted after the previous schedule is removed

### PR DESCRIPTION
**Pull request by rift137**

### CODE REVIEW

File: app/views/person_email_mailer/rescheduling_the_vault.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Hello <%= @new_owner_first_name %>,
+</br>
```
Comment (halloffame) [inline app/views/person_email_mailer/rescheduling_the_vault.html.erb]: In the html5 spec it is `<br>`. Otherwise it is `<br/>`.

Comment (halloffame) [inline app/views/person_email_mailer/rescheduling_the_vault.html.erb]: You could also just use `<p></p>` tags...

File: app/mailers/person_email_mailer.rb [hunk 2]
```diff
@@ -12,10 +12,28 @@ def changed_password(uuid: nil, address: nil, password: nil)
...
+    @vault_date = vault_date.to_date.strftime('%A %m/%d/%y')
+    @reminder_date = (vault_date.to_date - 15.days).strftime('%A %m/%d/%y')
+    @new_owner = Person.find_by(uuid: uuid)
+    @new_owner_first_name = @new_owner.first_name
+    @new_owner_email = @new_owner.email.address
+
+    return unless email = email_for(uuid).presence
+    mail(to: email, subject: '[Information Technology] Account Access Granted',
+         reply_to: 'it.helpdesk@biola.edu')
+  end
```
Comment (halloffame) [inline app/mailers/person_email_mailer.rb]: This looks like it is doing a lot of the same stuff as `rescheduling_the_vault`. Is there a way to share code between the two methods?

**Comment by rift137** [comment]
[Deleting a persons deprovision task should be able to be recreated without error.](https://trello.com/c/KGxWxdP9/636-deleting-a-persons-deprovision-task-should-be-able-to-be-recreated-without-error)

File: app/mailers/person_email_mailer.rb (hunk 2 above)
Comment (rift137) [inline app/mailers/person_email_mailer.rb]: done

**Pull request merged**